### DISCUSSION TITLE
Cannot import data_integrity

### DISCUSSION THREAD

**Discussion by ntvcuong** [deepchecks/deepchecks#1823] [Q&A] [answered]
Hi team, thanks for the useful and practical package. I love the idea but cannot use the library. After installing using bash script
`conda install -c conda-forge deepchecks`, I tried to import the module tabular with `from deepchecks.tabular.suites import data_integrity`, but an error appears `ImportError: cannot import name 'task_type_validation' from 'deepchecks.utils.metrics'`.
Can you please give some suggestions? Thanks

**Comment by noamzbr** [comment, marked as answer]
Hi @ntvcuong. It seems like there is something wrong with the installation, as `task_type_validation` was removed and is no longer part of the latest release that you can get from conda.

First thing you could try is to re-install deepchecks. Can you please try uninstalling the package - `conda remove deepchecks`. Would also try to remove it with pip just to be sure `pip uninstall deepchecks`. Then re-install deepchecks using the same conda command you used beforehand.

If you continue getting this error, could you please send us a freeze of your environment using `conda list`?

**Comment by ntvcuong** [comment]
hi @noamzbr , thanks for your suggestions. I can get it fixed now.
